spec/dummy/.gitignore

```diff
@@ -37,5 +37,5 @@ yarn-debug.log*
 *.bs.js
 
 # File Generated by ROR FS-based Registry
-/client/app/packs/server-bundle-generated.js
+/client/app/server-bundle-generated.js
```

Comment:
`/client/app/generated/server-bundle-generated.js`

and the generated directory is ignored...

Reply:
this is inconsistent with https://github.com/shakacode/react_on_rails/pull/1531/files#diff-cb9a4038d8838c04798d3da7dc2b785dffe109b7f9b903e354601e5cd5a9efe6R39